README.md

```diff
@@ -27,6 +41,17 @@ Perform an HTTP POST request and print the response:
 KnowSheet> HTTP(POST("http://httpbin.org/post", "BODY", "text/plain")).body
 ```
 
+Parse a JSON response into an object, get a field from it:
+```
+KnowSheet> JSONParse(HTTP(GET("http://httpbin.org/get?query=1")).body).args
+```
+
+POST a JSON-encoded object:
+```
+KnowSheet> JSONParse(HTTP(POST("http://httpbin.org/post", DemoObject())).body).json
+```
+<sup>(in C++, you'll have a cerealizable object in place of the `DemoObject()`)</sup>
```

Comment:
I'd stress that the C++ syntax is the same. Ex. <sup>The syntax mimics C++ Bricks exactly, as long as `DemoObject` is defined as a serializable type.</sup>.